session.go

```diff
@@ -287,6 +296,28 @@ func WithServer(addr string) ConnectOption {
 	}
 }
 
+// WithAdditionalServers configures the network address to dial to connect to the ngrok
+// service on secondary legs. Use this option only if you are connecting to a custom agent
+// ingress, and have enabled multi leg.
+//
+// See the [server_addr parameter in the ngrok docs] for additional details.
+//
+// [server_addr parameter in the ngrok docs]: https://ngrok.com/docs/ngrok-agent/config#server_addr
+func WithAdditionalServers(addrs []string) ConnectOption {
+	return func(cfg *connectConfig) {
+		cfg.AdditionalServerAddrs = addrs
```

Comment:
Would it be too surprising to `cfg.EnableMultiLeg = true` here too?

Reply:
hrm, I think I'd lean towards keeping it an explicit on or off. I can imagine user code where the addressing is set by a different part of the codebase than something doing the multi-leg decision, and that indeed might be a surprising side-effect to have it auto-enable itself.